features-json/avif.json

```diff
@@ -509,9 +509,8 @@
   "notes":"",
   "notes_by_num":{
     "1":"Can be enabled in Firefox via the `image.avif.enabled` pref in `about:config`.",
-    "2":"Only supports still images. AVIF sequences are not supported!",
-    "3":"Only available on macOS 13 Ventura or later",
-    "4":"Does not support images with noise synthesis."
+    "2":"Supports still images. Animated image sequences are not supported.",
```

Comment:
This note is currently only used for Firefox. Should it be set for Safari 16.0 as well?

Reply:
@mathiasbynens but it is on Safari for iOS 16.0 and since Safari (for macOS) 16.0 doesn't support AVIF at all, it doesn't need to be there?

![image](https://user-images.githubusercontent.com/2644614/190587698-568efd66-4c2a-49a7-8433-5bacca610085.png)

Reply:
Ah, I see, my bad!